grails-app/domain/org/pih/warehouse/inventory/InventoryLevel.groovy

```diff
@@ -9,10 +9,13 @@
  **/
 package org.pih.warehouse.inventory
 
+import grails.databinding.BindUsing
+import org.pih.warehouse.EmptyStringsToNullBinder
 import org.pih.warehouse.core.Location
 import org.pih.warehouse.product.Product
 import util.InventoryUtil
 
+@BindUsing(EmptyStringsToNullBinder)
```

Comment:
@alannadolny do you want to use this binder on every property or on a specific one?